### PR TITLE
Uses modern node.js in dockerfile

### PR DESCRIPTION
**Pull request by umaar**

### CODE REVIEW

File: docker/base/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,8 @@
 FROM ministryofjustice/ruby:2.5.1
 
+# https://github.com/nodesource/distributions/blob/master/README.md#installation-instructions
+RUN curl -sL https://deb.nodesource.com/setup_12.x | bash -
+
 RUN apt-get update && apt-get install -y nodejs postgresql-contrib libpq-dev
```
Comment (asmega) [inline docker/base/Dockerfile]: do we need to remove nodejs from this line?

Comment (umaar) [reply docker/base/Dockerfile]: From the installation instructions look like we should do both, I guess the first one is configuring the sources list 

image

Comment (solidgoldpig) [reply docker/base/Dockerfile]: But surely we should rewrite in Ruby.....

Comment (solidgoldpig) [reply docker/base/Dockerfile]: ^^^ I am being serious, just in case that comes across as snark

Comment (asmega) [reply docker/base/Dockerfile]: maybe its not even needed. it might have to do with asset minification, but I'd have to check

**Pull request merged**